Assets/MixedRealityToolkit-SDK/Inspectors/UX/Controllers/AttachToControllerInspector.cs

```diff
@@ -1,25 +1,24 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.﻿
 
-using Microsoft.MixedReality.Toolkit.SDK.UX.MotionController;
+using Microsoft.MixedReality.Toolkit.SDK.UX.Controllers;
 using UnityEditor;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.SDK.Inspectors.UX.MotionController
+namespace Microsoft.MixedReality.Toolkit.SDK.Inspectors.UX.Controllers
 {
     [CustomEditor(typeof(AttachToController))]
-    public class AttachToControllerInspector : ControllerFinderInspector
+    public class AttachToControllerInspector : Editor
     {
         private SerializedProperty positionOffsetProperty;
         private SerializedProperty rotationOffsetProperty;
         private SerializedProperty scaleOffsetProperty;
         private SerializedProperty setScaleOnAttachProperty;
         private SerializedProperty setChildrenInactiveWhenDetachedProperty;
 
-        protected override void OnEnable()
-        {
-            base.OnEnable();
 
+        protected void OnEnable()
```

Comment:
Why protected if you're not making it virtual?

If no virtual then we can just make it private.